projects/aca-folder-rules/src/lib/rule-details/actions/rule-action.ui-component.scss

```diff
@@ -3,5 +3,9 @@
     display: flex;
     flex-direction: row;
     gap: 20px;
+
+    adf-card-view .adf-property-list .adf-property adf-card-view-item-dispatcher adf-card-view-textitem .mat-form-field .mat-form-field-infix {
+      width: 280px;
```

Comment:
Should we be using an absolute value for this? Is there not a way to have it occupy the rest of the white space to the right? This might risk adding a pointless scroll bar on smaller screen widths

Reply:
hi @TomikaArome I checked by giving values in percentage, but it was not taking the percentage value and  mat-form-field-infix value that was setting earlier was 180px, so I changed that and as far as for scroll bar, even earlier scroll was coming in smaller screen widths, so it hasn't changed.

![Screenshot from 2023-03-14 18-39-55](https://user-images.githubusercontent.com/105338943/225011023-3b48da93-c779-4f5a-aa7f-84081d995df1.png)

Reply:
Ok I'll approve for now